tests/test_source.py

```diff
@@ -0,0 +1,118 @@
+# pylint: disable=redefined-outer-name
+import typing as tp
+
+import attr
+import plumbum as pb
+import pytest
```

Comment:
Codacy found an issue: [Unable to import 'pytest'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)